### PR TITLE
Fix MC-123848

### PR DESCRIPTION
**Pull request by BillyGalbreath**

### CODE REVIEW

File: patches/server/0920-Fix-MC-123848.patch [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+diff --git a/src/main/java/net/minecraft/world/entity/decoration/ItemFrame.java b/src/main/java/net/minecraft/world/entity/decoration/ItemFrame.java
+index d2a77b4ca343d19e1c70afe3f3906a9bd53d0eec..e2e9777b03bd786907ceaa7d773dd177fda28eff 100644
+--- a/src/main/java/net/minecraft/world/entity/decoration/ItemFrame.java
++++ b/src/main/java/net/minecraft/world/entity/decoration/ItemFrame.java
+@@ -282,6 +282,13 @@ public class ItemFrame extends HangingEntity {
+         }
+     }
+ 
++    // Purpur start
```
Comment (NoahvdAa) [inline patches/server/0920-Fix-MC-123848.patch]: Average hacktoberfest PR, blatantly stealing patches from other forks, thinking we wouldn't realize, smh...
```suggestion
+    // Paper start
```

**Review by NoahvdAa — APPROVED** [review]

**Comment by Lulu13022002** [comment]
We should probably wait for https://github.com/PaperMC/Paper/pull/8187 and then patch in it ?

**Comment by Lulu13022002** [comment]
We should probably wait for https://github.com/PaperMC/Paper/pull/8187 and then patch in it ?

**Comment by BillyGalbreath** [comment]
> We should probably wait for #8187 and then patch in it ?

Doesn't seem fair to the contributor to merge someone's work into someone else's patch like that. :-1: from me.

**Comment by NoahvdAa** [comment]
> > We should probably wait for #8187 and then patch in it ?
> 
> Doesn't seem fair to the contributor to merge someone's work into someone else's patch like that. 👎 from me.

You'd still be added as a coauthor to the patch

**Review by Owen1212055 — APPROVED** [review]

**Pull request closed without merging**